docs/kb/deploy-event-router-kind.md

```diff
@@ -0,0 +1,315 @@
+
+```console
+cat << EOF | kubectl create -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vcsim
+spec:
+  selector:
+    matchLabels:
+      app: vcsim
+  template:
+    metadata:
+      labels:
+        app: vcsim
+    spec:
+      containers:
+        - name: vcsim
+          image: vmware/vcsim:latest
+          args: ["/vcsim", "-l", ":8989"]
+          ports:
+            - name: https
+              containerPort: 8989
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vcsim
+spec:
+  selector:
+    app: vcsim
+  ports:
+    - name: https
+      port: 443
+      targetPort: 8989
+EOF
+
+deployment.apps/vcsim created
+service/vcsim created
+```
+
+## Install `govc`
+
+[`govc`](https://github.com/vmware/govmomi/tree/master/govc) is used to perform
+operations against the (simulated) vCenter, e.g. powering off a virtual machine
+which will trigger a corresponding event.
+
+```console
+brew install govc
+
+govc about
+govc: specify an ESX or vCenter URL
+```
+
+## Install `Helm`
+
+The easiest way to deploy the VMware Event Router is via
+[Helm](https://helm.sh/).
+
+```console
+brew install helm
+
+helm version
+version.BuildInfo{Version:"v3.5.4", GitCommit:"1b5edb69df3d3a08df77c9902dc17af864ff05d1", GitTreeState:"dirty", GoVersion:"go1.16.3"}
+```
+
+## Add the VMware Event Router Helm Repo
+
+```console
+helm repo add vmware-veba https://projects.registry.vmware.com/chartrepo/veba
+
+# update index in case the repo was already installed
+helm repo update
+
+# ignore the index error message which is due to the Harbor environment
+helm search repo event-router
+index.go:339: skipping loading invalid entry for chart "event-router" "release-0.5.0" from /Users/mgasch/Library/Caches/helm/repository/vmware-veba-index.yaml: validation: chart.metadata.version "release-0.5.0" is invalid
+NAME                            CHART VERSION   APP VERSION     DESCRIPTION
+vmware-veba/event-router        v0.6.5          development     The VMware Event Router is used to connect to v...
+```
+
+## Deploy VMware Event Router
+
+Before installing the VMware Event Router create a configuration (Helm
+"override") file (passed via `-f -` and read from stdin).
+
+The following commands assume a Knative `broker` in the `default` Kubernetes
+namespace with the name `example-broker`. As of writing this document these are
+the values used by the KonK stack. You can verify this via:
+
+```console
+kubectl get brokers -A
+NAMESPACE   NAME             URL                                                                               AGE   READY   REASON
+default     example-broker   http://broker-ingress.knative-eventing.svc.cluster.local/default/example-broker   28m   True
+```
+
+It is also assumed that you created the `vcsim` deployment as shown above,
+otherwise change the address and settings if you have done a custom deployment.
+
+```console
+# create the deployment "router" in the "vmware" namespace
```

Comment:
nit: can we use `vmware-system` namespace to align with VEBA? (change all occurrences)